gradle/libs.versions.toml

```diff
@@ -4,13 +4,17 @@ managed-swagger = "2.2.7"
 managed-swagger-compat = "2.2.7"
 managed-javadoc-parser = "0.3.1"
 managed-jsoup = "1.15.3"
-managed-jakarta-validation-api = "3.0.2"
```

Comment:
if we were already managing jakarta we should not remove it. That could be a breaking change for users' applications.

Reply:
@sdelamo ok. I reverted it. But it was mistake. We don't need to manage this dependency too